src/lib/Components/Home/HeroUnit.tsx

```diff
@@ -0,0 +1,86 @@
+import { isPad } from "lib/utils/hardware"
+import { useScreenDimensions } from "lib/utils/useScreenDimensions"
+import { Flex, Text, useColor } from "palette"
+import React, { useState } from "react"
+import { Image, TouchableOpacity, View } from "react-native"
+import LinearGradient from "react-native-linear-gradient"
+
+interface Props {
+  unit: any
```

Comment:
Let's try to avoid the use of `any` whenever possible. To get the correct type from Relay you can use:

```suggestion
  unit: NonNullable<NonNullable<HomeHero_homePage["heroUnits"]>[0]>
```